lib/player.rb

```diff
@@ -12,6 +13,7 @@ def initialize(display)
   end
 
   def selection(board)
+    display.output(MESSAGES[:player_prompt] + board.next_token)
```

Comment:
Is there a way we could make this more intention revealing?

Reply:
Sure